app/assets/js/rtMedia.backbone.js

```diff
@@ -8,6 +8,31 @@ var rtmedia_load_template_flag = true;
 
 jQuery( function( $ ) {
 
+	/**
+	 * Issue 1059 fixed: negative comment count
+	 */
+	$( document ).ready( function () {
+		/**
+		 * Bind dynamic event on delete button to remove media ul
+		 */
+		$( '#activity-stream' ).on( 'click', '.acomment-delete', function () {
+			/**
+			 * get media ul
+			 */
+			let media_children = $( this ).parent().parent().find( 'div.acomment-content ul.rtmedia-list' );
```

Comment:
instead of `parent().parent()` please use `closest();`
https://api.jquery.com/closest/